lib/debezium/types_test.go

```diff
@@ -284,14 +284,14 @@ func TestField_ParseValue(t *testing.T) {
 			field := Field{Type: Int64, DebeziumType: MicroTime}
 			value, err := field.ParseValue(int64(54720000000))
 			assert.NoError(t, err)
-			assert.Equal(t, "15:12:00.000000", value.(*ext.ExtendedTime).String(""))
+			assert.Equal(t, "15:12:00.000000", value.(*ext.ExtendedTime).GetTime().Format("15:04:05.000000"))
```

Comment:
Could we use `MicroTime{}.layout()` here instead of hardcoding the layout?

Reply:
I think we'll run into a cyclical dependency import since it's not in the same package :(